Resources/Content/MSGraph/Applications/Simeon Cloud Sync.json

```diff
@@ -16,9 +16,7 @@
     "preAuthorizedApplications": [

```

Comment:
Would you be willing to reformat this file so it doesn't show as a change in a future PR?